RecoLocalCalo/HGCalRecAlgos/src/HGCalImagingAlgo.cc

```diff
@@ -197,7 +197,7 @@ math::XYZPoint HGCalImagingAlgo::calculatePosition(std::vector<KDNode> &v){
 			 z/total_weight );
     }
   }
-  else {
+  else if (v_size > 0) {
```

Comment:
but then maybe I missed the logic and maxEnergyIndex=0 even if it's not modified in the loop is meant to represent a valid value.
Ignore my last comment in this case.